src/test/java/LoginTests.java

```diff
@@ -0,0 +1,19 @@
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+import pages.HomePage;
+import pages.LoginPage;
+
+public class LoginTests extends BaseTest {
+    @Test
+    public void LoginValidEmailPasswordTest() {

```

Comment:
Methods should be verbs, in mixed case with the first letter lowercase, with the first letter of each internal word capitalized.

For example `public static void loginEmptyEmailPasswordTest ()`